File: js/task1/S1_VAR_05.js
function isArray(value) {
  return Array.isArray(value);
}

console.log(isArray([]));              
console.log(isArray([1, 2, 3]));       
console.log(isArray(new Array()));     

console.log(isArray({}));              
console.log(isArray(null));            
console.log(isArray(undefined));       
console.log(isArray(42));              
console.log(isArray("array"));         
console.log(isArray(() => {}));        
console.log(isArray(Symbol()));

File: python/task1/S1_VAR_05.py
def isArray(value):
  return isinstance(value, list)

print(isArray([]))
print(isArray([1, 2, 3]))
print(isArray(list()))

print(isArray({}))
print(isArray(None))
print(isArray(None))
print(isArray(42))
print(isArray("array"))
print(isArray(lambda: None))
print(isArray(object()))
